operations/customers.md

```diff
@@ -357,169 +404,198 @@ Returns all open items of the specified customers, i.e. all unpaid items and all
+| `DriversLicense` | [Document](customers.md#document) | optional | Drivers license details of the customer. |
+| `Classifications` | array of [Customer classification](customers.md#customer-classification) | optional | Classifications of the customer. |
+| `Options` | array of [Customer option](customers.md#customer-option) | optional | Options of the customer. |
+| `ChainId` | string | optional | Unique identifier of the chain. Required when using `PortfolioAccessTokens`, ignored otherwise. |
+| `OverwriteExisting` | boolean | required | Whether an existing customer should be overwritten in case of duplicity. This applies only to basic personal information (`Title`, `FirstName`, `LastName`, ...). |
+| `ItalianDestinationCode` | string | optional | Value of Italian destination code. |
+| `ItalianFiscalCode` | string | optional | Value of Italian fiscal code. |
 
-```javascript
-{
-    "Customers": [
-        {
-            "CustomerId": "2a1a4315-7e6f-4131-af21-402cec59b8b9",
-            "OrderItems": [
-                {
-                    "Id": "35820535-b988-4d6f-80cf-ecb19cdc3e58",
-                    "AccountId": "2a1a4315-7e6f-4131-af21-402cec59b8b9",
-                    "OrderId": "6645113e-72cc-4f99-ab77-e4452911bf20",
-                    "BillId": null,
-                    "AccountingCategoryId": "d250149e-a29d-4c70-b607-a1759faf7320",
-                    "Amount": {
-                        "Currency": "GBP",
-                        "NetValue": -5.00,
-                        "GrossValue": -5.00,
-                        "TaxValues": [
-                            {
-                                "Code": "UK-Z",
-                                "Value": 0.0
-                            }
-                        ],
-                        "Breakdown": {
-                            "Items": [
-                                {
-                                    "TaxRateCode": "UK-Z",
-                                    "NetValue": -5.00,
-                                    "TaxValue": 0.0
-                                }
-                            ]
-                        }
-                    },
-                    "RevenueType": "Additional",
-                    "ConsumedUtc": "2017-04-04T15:13:39Z",
-                    "ClosedUtc": null,
-                    "AccountingState": "Open",
-                    "Data": {
-                        "Discriminator": "Rebate",
-                        "Value": {
-                            "RebatedItemId": null
-                        }
-                    }
-                }
-            ],
-            "PaymentItems": []
-        }
-    ]
-}
-```
+#### Sex
 
-| Property | Type | Contract | Description |
-| :-- | :-- | :-- | :-- |
-| `Customers` | array of [Customer items](#customer-items) | required | The customers with their items. |
+* `Male`
+* `Female`
 
-#### Customer items
+#### Document
```

Comment:
issue: This is confusing. We already have Identity document which was previously a Document. I think the difference here is that this is an add/update parameter without ID.